package_manager/contracts/PR/RepoPackage.sol

```diff
@@ -0,0 +1,98 @@
+pragma solidity ^0.4.24;
+
+
+/**
+ * @title RepoPackage
+ */
+contract RepoPackage {
+  event NewVersion(uint256 versionId, uint16[3] semanticVersion);
+
+  struct Version {
+    uint16[3] semanticVersion;
```

Comment:
I'd consider using three `uint64`. I know it sounds like a lot but I don't see a reason for using smaller numbers.

Reply:
Actually using uint64 allows for alternative version identifiers, such as time-based versions (20180925 does not fit in 16 bits), so +1 to this change.

Reply:
I like that, this should not be a problem for Aragon since it is an array right? I mean, we won't need to perform any migrations or so, right?